apps/web/components/SignInModal.tsx

```diff
@@ -0,0 +1,52 @@
+import { Input } from "antd";
+import { useTranslation } from "react-i18next";
+import { IoIosCloseCircle } from "react-icons/io";
+
+import { PrimaryButton } from "ui";
+import {
+  Container,
+  ContentModal,
+  TitleContent,
+  Title,
+  SubTitle,
+  DataField,
+  CloseButton,
+} from "../styles/components/ModalStyles";
+
+interface ModalProps {
+  signInModal: Boolean;
+  handleSignInModalVisibility: () => void;
+}
+
+export default function SignInModal({ signInModal, handleSignInModalVisibility }: ModalProps) {
```

Comment:
@MFelipeSilva why do we need these props? we can just create a component called Modal instead of creating a SignInModal, SignUpModal, etc ...